lizmap/modules/view/controllers/translate.classic.php

```diff
@@ -36,6 +36,8 @@ public function index()
         }
 
         $data = LocalesLoader::getLocalesFrom('view~dictionnary', $lang);
+        $fallback = LocalesLoader::getLocalesFrom('view~dictionnary', \jApp::config()->fallbackLocale);
```

Comment:
Please enclose the new lines into this test : `if (strpos(\jApp::config()->jResponseHtml['plugins'], 'debugbar') !== false) {`. So it will be executed only in the "development mode".